### ISSUE TITLE
Accessibility - using the ESC key while keyboard focus is on the toaster should close the toaster for all types

### ISSUE THREAD

**Issue by ahayes91** [diegoddox/react-redux-toastr#260]
From the docs, I saw that the "confirm" type of toaster message caters for closing the toaster on ESC:
```
Toastr: confirm
The confirm method takes two arguments. The first one is the message and the second one is an object where you can specify what will happen when the user clicks on ok or on the cancel buttons or on keypress enter/esc.
```

Ideally this should be an option for all types of toasters - when focus is on the toaster, you should be able to use the ESC key to close it, as per accessibility standards for modal dialogs: https://bitsofco.de/accessible-modal-dialog/

**Comment by ahayes91** [comment]
Looks like I could do something similar to this https://github.com/diegoddox/react-redux-toastr/issues/244#issuecomment-629123984 and just add an ESC key handler to the X button - in my case that's enough, because the only focusable element in our toaster is the X button. However, it is still worth fixing properly with a prop in case other focusable elements are included in the toaster for any reason.

**Comment by diegoddox** [comment]
Hi @ahayes91 thanks for the link, I will take a look on it.
